drizzlepac/runastrodriz.py

```diff
@@ -334,16 +334,17 @@ def process(inFile,force=False,newpath=None, inmemory=False, num_cores=None,
             # Create trailer marker message for start of align_to_GAIA processing
             _trlmsg = _timestamp("Align_to_GAIA started ")
             print(_trlmsg)
-            ftmp = open(_tmptrl,'w')
+            ftmp = open(_tmptrl, 'w')
             ftmp.writelines(_trlmsg)
             ftmp.close()
-            _appendTrlFile(_trlfile,_tmptrl)
+            _appendTrlFile(_trlfile, _tmptrl)
             _trlmsg = ""
 
             # Create an empty astropy table so it can be used as input/output for the perform_align function
-            #align_table = Table()
+            # align_table = Table()
```

Comment:
We can get rid of Line 344 now as we do not want to mislead someone reading the code.  It was part of the experimenting with return values when the logging decorator caused the return value to be absorbed.

Reply:
Fix included in 2d3949e .